build/media_source/plg_system_shortcut/shortcut.js

```diff
@@ -0,0 +1,59 @@
+function handleKeyPressEvent(e) {
+    console.log(e);
+    // On Press CTRL + S
+    if (
+      (window.navigator.platform.match("Mac") ? e.metaKey : e.ctrlKey) &&
```

Comment:
`(window.navigator.platform.match("Mac") ? e.metaKey : e.ctrlKey)`

Common code put in common place

Reply:
I will update it. Thank you